extensions/amp-analytics/0.1/events.js

```diff
@@ -453,6 +452,14 @@ export class AmpStoryEventTracker extends CustomEventTracker {
   fireListener_(event, rootTarget, config, listener) {
     const type = event['type'];
     const vars = event['vars'];
+    const storySpec = config['storySpec'] || {};
+    const repeat =
+      storySpec['repeat'] === undefined ? true : storySpec['repeat'];
```

Comment:
Yeah mostly to not change the current behavior. We could change the name of the option to be something like `blockRepeat` / `skipRepeat` / `sendOnce` / ... and make the default value be false. WDYT? @zhouyx @gmajoulet do you have any preferences?

Reply:
In terms of naming, I don't have a strong preference. I asked about the default behavior because  I feel most people would have `repeat : false` setting, because I don't think it should be counted as multiple pageview if one navigate back and forth. Let me know if you don't think that's the case. 
Is there an event to collect the view of a story? not a single story page? 

I also understand we have to stick with the default value to not break current user. I'm good with this.

Reply:
we don't have an event to collect the view of a story. Maybe using the general AMP trigger `pageview`

Reply:
just my personal opinion that one may not want to collect duplicate pageview if one navigate within the same story. 

I'm fine with the current behavior as long as it is well communicated : )

Reply:
Sounds good, thanks!